utils/wasm-instrument/tests/fixtures/stack-height/empty_functions.wat

```diff
@@ -0,0 +1,8 @@
+(module
```

Comment:
and these wats are pretty small. dont we want to inline them in test code to see both in/out in place?

Reply:
Some of them is 100 lines long so I'd leave them as is

Reply:
ok